test/test_encryption.py

```diff
@@ -2335,6 +2335,31 @@ def test_02_success(self):
         self.client_encryption.create_data_key("aws", self.master_key)
 
 
+# https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#8-bypass-spawning-mongocryptd
+class TestBypassMongocryptWithSharedLibrary(EncryptionIntegrationTest):
+    @unittest.skipUnless(os.environ.get("TEST_CRYPT_SHARED"), "crypt_shared lib is not installed")
+    def test_via_loading_shared_library(self):
+        schemas = {"db.,coll": json_data("external", "external-schema.json")}
+        opts = AutoEncryptionOpts(
+            KMS_PROVIDERS,
+            "keyvault.datakeys",
+            schema_map=schemas,
+            mongocryptd_uri="mongodb://localhost:27021/db?serverSelectionTimeoutMS=1000",
+            mongocryptd_spawn_args=[
+                "--pidfilepath=bypass-spawning-mongocryptd.pid",
+                "--port=27021",
```

Comment:
This port is inconvenient since it's used by launch when starting a sharded cluster. Let's use 47021 instead.